core/src/main/scala/com/salesforce/op/stages/impl/selector/ModelSelector.scala

```diff
@@ -143,13 +144,14 @@ E <: Estimator[_] with OpPipelineStage2[RealNN, OPVector, Prediction]]
   final override def fit(dataset: Dataset[_]): SelectedModel = {
 
     implicit val spark = dataset.sparkSession
+    JobGroupUtil.setJobGroup(OpStep.CrossValidation)
```

Comment:
can `setJobGroup` return the previous job group? that way you can reset it back:
```scala
val prevJobGroup = JobGroupUtil.setJobGroup(OpStep.CrossValidation)
// a lot of code
JobGroupUtil.setJobGroup(prevJobGroup)
```

Reply:
It can't, nor is there a 'getJobGroup`.

I could implement it myself, with `JobGroupUtil.setTemporaryJobGroup()`, but this is the only place and cross-validation is kind of a special case anyway.